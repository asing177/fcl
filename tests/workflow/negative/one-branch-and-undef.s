global int x;

@initial
split()
{
  transitionTo(@{a1,b});
}

@a1
init()
{
  x = 0;
  transitionTo(@a2);
}

@{a2,b}
join()
{
  transitionTo(@commonState);
}

@initial
noop()
{
  transitionTo(@commonState);
}

@commonState
use()
{
  y = x;
  terminate();
}
