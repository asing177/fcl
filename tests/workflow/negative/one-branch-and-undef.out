Variable "x" undefined at:
  - line 31:7
Variable "y" undefined at:
  - line 31:3
Stack trace leading up to error:
   noop : {initial} -> {commonState}
   use : {commonState} -> {terminal}
